onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -723,7 +723,21 @@
             or (tensor_name in self.generated_value_names)
         )
 
```

Comment:
>False, False, False [](http://example.com/codeflow?start=53&length=19)

Three 'False' in a roll, a little hard to read, consider:
initializer_use_weight_qType=False, reduce_range=False, op_level_per_channel=False